prow/scripts/cluster-integration/helpers/get-letsencrypt-cert.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Please fix formatting is this file

Add a description to this script.

Add `set -o errexit`